src/Microsoft.PowerShell.Commands.Utility/commands/utility/Tee-Object.cs

```diff
@@ -85,6 +86,16 @@ public string Variable
 
         private string _variable;
 
+        /// <summary>
+        /// Sets the encoding to use when writing the file.
```

Comment:
```suggestion
        /// Gets and sets the encoding to use when writing the file.
```

Reply:
Applied the suggestion, but it seems like that didn't fix the codeclimate issue.